install-dev/data/db_structure.sql

```diff
@@ -2139,7 +2139,7 @@ CREATE TABLE `PREFIX_webservice_permission` (
   `id_webservice_permission` int(11) NOT NULL AUTO_INCREMENT,
   `resource` varchar(50) NOT NULL,
   `method` enum(
-    'GET', 'POST', 'PUT', 'DELETE', 'HEAD'
+    'GET', 'POST', 'PUT', 'PATCH', 'DELETE', 'HEAD'
```

Comment:
Could you create a PR on `autoupgrade` for the migration ? (Not blocking this PR)

Reply:
https://github.com/PrestaShop/autoupgrade/pull/479

Reply:
Fixed.